### ISSUE TITLE
Tooltip broken in Windows

### ISSUE THREAD

**Issue by sjoshid** [linebender/druid#1986]
Tooltips were introduced in #1919. But they aren't working correctly in Windows specifically when you set scaling to something other than 100%. Some issues (seen in sub_window.rs example) are:

* Tooltips open up in a different location.
* The closable/drag-able window is broken as well.

**Comment by giannissc** [comment]
In case you can't tell the mouse pointer is over the label widget. After the update the full screen hovering works great and the tooltip is right under the mouse pointer however for the unmaximised windows, the tooltip more the further away the further you more from the screen's left corner

**Comment by giannissc** [comment]
But weren't these issues supposed to be solved with https://github.com/linebender/druid/pull/1982?

**Comment by sjoshid** [comment]
Not really. There was no set definition of what `set_position` took: you could pass any type of `Point`.

With the PR, definition of `set_position` is clear: if creating a window with a parent window (tooltips or dropdowns) you _must_ pass a point in parent window coord.

**Comment by giannissc** [comment]
Does it work out of the box with tooltip code or did you have to make some changes? Cause it doesn't work right for me..
